pkg/manifest/manifest_loader.go

```diff
@@ -248,39 +221,65 @@ func parseOAuth(a oAuth) (OAuth, error) {
 		return OAuth{}, fmt.Errorf("failed to parse ClientSecret: %w", err)
 	}
 
+	var urlDef UrlDefinition
+	if (a.TokenEndpoint == url{}) {
+		urlDef = UrlDefinition{
+			Value: client.OAuthTokenEndpoint,
+		}
+	} else if urlDef, err = parseUrlDefinition(a.TokenEndpoint); err != nil {
+		return OAuth{}, fmt.Errorf("failed to parse \"tokenEndpoint\": %w", err)
+	}
+
 	return OAuth{
-		ClientId:     clientID,
-		ClientSecret: clientSecret,
+		ClientId:      clientID,
+		ClientSecret:  clientSecret,
+		TokenEndpoint: urlDef,
 	}, nil
 }
 
-func parseManifestFile(context *ManifestLoaderContext, data []byte) (manifest, []error) {
-	var m manifest
-	var errs []error
+func getManifestYAML(context *ManifestLoaderContext) (manifest, error) {
```

Comment:
The idea was to report all issues we can find at once to a user. This change now reports only one error, the user fixes it, and then the next error and so on. Why did you change it?

Also the name does not fit - I think parseManifest was quite good.

Reply:
`getManifestYAML` just checks does file exists and is it a readable and if can, call unmarshall function

Logic from `parseManifestFile` is part of `verifyManifestYAML` which returns `[]error`. 

`getManifestYAML` is renamed to `readManifestYAML`